src/odemis/driver/test/xt_client_test.py

```diff
@@ -1462,5 +1477,256 @@ def test_apply_autostigmator(self):
         autostigmator_future.result(timeout=30)
 
 
+class TestCheckLatestPackage(unittest.TestCase):
+    """
+    Test the check_latest_package function.
+
+    Folders/files for testing:
+        delmic-fastem-xtadapter-64bit-v1.11.2
+        delmic-xtadapter-32bit-v1.11.2
+        delmic-xtadapter-32bit-v1.11.2-dev.zip
+
+    """
+
+    def setUp(self):
+        self.dir = "/tmp/"
+        self.dir_names = [
+            "delmic-fastem-xtadapter-64bit-v1.11.2",    # has 1 .exe file
+            "delmic-xtadapter-32bit-v1.11.2",           # has 1 .exe file
+            "delmic-fastem-xtadapter-64bit-v2.11.2",    # has 0 .exe file
+            "delmic-xtadapter-32bit-v2.11.2",           # has 0 .exe file
+            "delmic-fastem-xtadapter-64bit-v3.11.2",    # has 2 .exe file
+            "delmic-xtadapter-32bit-v3.11.2",           # has 2 .exe file
+        ]
+        for dir_name in self.dir_names:
+            if not os.path.isdir(os.path.join(self.dir, dir_name)):
+                os.makedirs(os.path.join(self.dir, dir_name))
```

Comment:
I think you can also use `os.makedirs(os.path.join(self.dir, dir_name), exist_ok=True)`